scripts/index.md

```diff
@@ -0,0 +1,42 @@
+# Available scripts index
+
+## Scripts and processes endpoints
+This index contains a list of all the provided scripts in DSpace REST API contract,
+to learn more about the scripts and processes endpoints; check the following documentation: 
+
+* [Processes](https://github.com/DSpace/RestContract/blob/main/processes-endpoint.md)
```

Comment:
The Processes & Scripts links in this file should be relative paths.  Currently they are full URLs to GitHub